pkgs/development/tools/rtags/default.nix

```diff
@@ -16,11 +16,13 @@ stdenv.mkDerivation rec {
            MACOSX_DEPLOYMENT_TARGET="10.9"
   '';
 
+  NIX_CFLAGS_COMPILE = "-fexceptions";
```

Comment:
On investigation, it looks like `-fno-exceptions` is coming from `$(llvm-config --cxxflags)` on line 11. If you add `-fexceptions` after that I'll approve these changes.